src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -74,7 +74,7 @@ export function getWarningHeadline(enrollmentStatus) {
     default:
       break;
   }
-  return <h4 className="usa-alert-heading">{content}</h4>;
+  return <h2 className="usa-alert-heading">{content}</h2>;
```

Comment:
Looks good, but it would be helpful if your screenshot showed how things look with this code change so we could see that everything looks in both the markup and the rendered page.